README.md

```diff
@@ -15,8 +15,8 @@ The transaction internal module is a dependency module which required for Baller
 
 ### Setting Up the Prerequisites
 
-1. Download and install Java SE Development Kit (JDK) version 11 (from one of the following locations).
-   * [Oracle](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
+1. Download and install Java SE Development Kit (JDK) version 17 (from one of the following locations).
+   * [Oracle](https://www.oracle.com/java/technologies/javase-jdk17-downloads.html)
```

Comment:
This link seems not valid

Reply:
I think we need to use https://www.oracle.com/java/technologies/downloads/#java17

Reply:
Lets add more generic link like https://www.oracle.com/java/technologies/downloads/